### PR TITLE
Update Github Gists Extension

### PR DESCRIPTION
**Pull request by Aayush9029**
## Description

- Update dependencies
- Added syntax highlighting

## Screencast

github-gist-1

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

**Review by pernielsentikaer — APPROVED** [review]

**Comment by pernielsentikaer** [comment]
Hi @Aayush9029 👋 

Awesome! Thanks for the enhancement of this extension 🔥

@koinzhang can you check this?

**Comment by koinzhang** [comment]
@Aayush9029 Thank you very much for your work, this extension has gotten even better!
But there is a problem, Markdown formatted files should not be included by the ''''' code block syntax, which would invalidate the Markdown syntax, as I have commented in the code

File: extensions/github-gist/src/util/utils.ts [hunk 1]
```diff
@@ -23,13 +23,12 @@ export const imgExt = [".svg", ".gif", ".jpg", ".jpeg", ".png"];
 
 export const getGistDetailContent = (gistFile: GistItem, gistFileContent: string) => {
   const gistFileExt = parse(gistFile.filename).ext;
+
   if (imgExt.includes(gistFileExt)) {
     return `![](${gistFile.raw_url})`;
   }
-  if (gistFileExt == ".md") {
```
Comment (koinzhang) [inline extensions/github-gist/src/util/utils.ts:29]: @Aayush9029 
Do not delete this code, please treat the Markdown format file separately: Do not use code block syntax for Markdown format gist

**Comment by koinzhang** [comment]
@pernielsentikaer LGTM

**Review by pernielsentikaer — APPROVED** [review]
Hi 👋 

Looks good to me, approved 🔥

**Pull request merged**